### ISSUE TITLE
Bug in task event-list commnand

### ISSUE THREAD

**Issue by poquirion** [globus/globus-cli#584]
```
$ globus version
Installed version:  3.1.2
Latest version:     3.1.2

You are running the latest version of the Globus CLI

$ globus task event-list  --filter-non-errors   6c978c8c-faf0-11eb-ab67-d195c983855c
TypeError: task_event_list() got an unexpected keyword argument 'filter'

```

I get the same thing with the `--filter-errors` option.

**Comment by sirosen** [comment]
Thanks for reporting this. It appears to be another issue with the transition to globus-sdk v3.0 (there have been a few of these, where our testing is weakest).
We'll get this resolved right away.

**Comment by sirosen** [comment]
We've just released v3.1.3 which fixes this. We'll also be notifying the listhost about this, but I figured we should let you know here as well.

Again, thanks for the bug report!
